frontend/src/components/homepage/homepage.jsx

```diff
@@ -0,0 +1,161 @@
+                <h1 className="text-slate-900 font-black text-5xl sm:text-4xl md:text-left lg:text-5xl tracking-tight text-center dark:text-white">
+                  Learn Anytime Anywhere.
+                </h1>
+                <p className="my-6 text-lg lg:text-xl text-slate-600 text-center md:text-left max-w-3xl mx-auto dark:text-slate-400">
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Officia ut consequuntur, optio sed vero vitae beatae nisi!
+                </p>
+              </div>
+              <div className="mt-6 md:mt-0 image md:w-[40%] md:order-1">
+                {/*The images here will be screenshots of the app to show users all the features*/}
+                <img src={section2Image} alt="learn english" />
+              </div>
+            </div>
+          </div>
+        </div>
+        {/*This section contains the two links to choose user */}
+        <div className="section3">
+          <div className="container px-5 m-auto lg:w-[960px] xl:w-[1140px] 2xl:w-[1320px] h-full">
+            <div className="w-full flex flex-wrap gap-[50px] md:gap-[10%] py-14 items-center justify-center">
+              <div className="role p-3 w-full max-w-[400px] md:w-[45%] rounded-[40px] bg-blue-400/50">
+                <div className="image md:w-full max-h-[200px] justify-center flex py-4">
+                  <img src={studentsImage} alt="help each other to learn" />
+                </div>
+                <div className="content w-full text-center">
+                  <h1 className="text-slate-900 font-black text-2xl tracking-tight text-center dark:text-white">
+                    Improve Your Language
+                  </h1>
+                  <p className="mt-6 text-lg text-slate-600 text-center max-w-3xl mx-auto dark:text-slate-400">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    Voluptas quae nostrum, vitae consequatur eaque quas rem!
+                  </p>
+                  <div className="w-full my-6 flex items-center justify-center">
+                    <Link to="/signup">
+                      <button className=" hover:bg-blue-800 bg-blue-700 focus:outline-none text-white font-semibold h-10 px-6 rounded-md flex items-center justify-center w-[12rem]">
+                        Start Learning
+                      </button>
+                    </Link>
+                  </div>
+                </div>
+              </div>
+              <div className="role p-3 w-full max-w-[400px] md:w-[45%] rounded-[40px] bg-[#fa974663]">
+                <div className="image md:w-full max-h-[200px] justify-center flex py-4">
+                  <img src={studentsImage} alt="help each other to learn" />
+                </div>
+                <div className="content w-full text-center">
+                  <h1 className="text-slate-900 font-black text-2xl tracking-tight text-center dark:text-white">
+                    Became A Tutor
```

Comment:
All looks good to me 🙂. Just a small typo to be-> 'Become A Tutor'

Reply:
typo fixed, merging PR